Overhaul about page

HereAround opened the pull request:
Preview:  https://herearound.github.io/oscar-website/

I just talked with @fieker on this PR, and @fieker seems happy with it. Pinging also @fingolfin .

HereAround committed:
Overhaul about page

fingolfin commented on about.md:
```diff
@@ -1,91 +1,91 @@
-unidirectional interfaces. Furthermore it serves as high-level
-language for implementing efficient algorithms utilizing all
-cornerstones.
+**OSCAR (Open Source Computer Algebra Resource)** is an open-source computer algebra system written in the [Julia programming language](https://julialang.org). It provides a unified, high-performance framework for computations in algebra, geometry, number theory, and polyhedral geometry. OSCAR builds on the capabilities of four major systems—[GAP](https://www.gap-system.org), [Singular](https://w ...
```
While some people claim OSCAR stands for "Open Source Computer Algebra Resource", the website and our manual all universally state "Open Source Computer Algebra Research system" (hence `oscar-system.org`).

HereAround commented:
Let me summarize conversation with @fingolfin on this PR:

Generally speaking @fingolfin seemed happy, in that this about page is better than the current one. But there are things that can be improved. In particular, I will add a brief description of the picture, to explain it to people. This is to emphasize that those are just *some* of the example applications of OSCAR, rather than all.

@fingolfin suggested to ping @thofma @fieker regarding the description/presentation of the number theory part. @fieker seemed reasonably happy with these changes, when we talked yesterday. However @fingolfin raised a few more questions, on which I need your input. Questions include:
* How to present the historic developments on ANTIC? Describe the historic developments? Avoid mentioning ANTIC and rather describe the Hecke-Nema-AbstarctAlgebra-Flint interface? A hybrid as currently is? Yet other?
* Is maybe Hecke "the" number theory package/oracle for OSCAR?
* Correspondingly, is AbstractAlgebra "the" foundational algebra building block for most of OSCAR?

If you can think of good answers to those questions, maybe you could propose how the text in this PR could be improved?

(For a preview, click the link in the first comment.)

HereAround committed:
More improvements on the about page

HereAround commented:
Caption added to the picture. Also changed the OSCAR abbreviation explanation to include *system*.

thofma committed:
try something

thofma commented:
I have pushed a suggestion, going via some hybrid approach. Keep "ANTIC" for historical reasons, but
- replace "ANTIC" by "ANTIC (Hecke)" in the picture (and later in the title)
- do not mention the antic C-library

I have also reordered the cornerstones as in the picture (which are ordered alphabetically).

fingolfin commented on about.md:
```diff
@@ -1,91 +1,93 @@
-unidirectional interfaces. Furthermore it serves as high-level
-language for implementing efficient algorithms utilizing all
-cornerstones.
+**OSCAR** (**O**pen **S**ource **C**omputer **A**lgebra **R**esearch System) is an open-source computer algebra system written in the [Julia programming language](https://julialang.org). It provides a unified, high-performance framework for computations in algebra, geometry, number theory, and polyhedral geometry. OSCAR builds on the capabilities of four major systems—[GAP](https://www.gap-system ...
```
Let's split this into somewhat shorter lines? Also how about starting like this:
```suggestion
**OSCAR** is an **O**pen **S**ource **C**omputer **A**lgebra **R**esearch System
written in the [Julia programming language](https://julialang.org). It
provides a unified, high-performance framework for computations in
algebra, geometry, number theory, and polyhedral geometry. OSCAR builds
on the capabilities of four major systems:
  [GAP](https://www.gap-system.org),
  [Singular](https://www.singular.uni-kl.de),
  [Polymake](https://polymake.org), and the now-retired
  [ANTIC](https://github.com/flintlib/antic),
whose functionality lives on through
  [Hecke](https://github.com/thofma/Hecke.jl),
  [Nemo](https://github.com/Nemocas/Nemo.jl), and
  [AbstractAlgebra](https://github.com/Nemocas/AbstractAlgebra.jl).
By combining these specialized tools under one umbrella, OSCAR enables
workflows and mathematical functionality that go far beyond what each system
offers on its own. *We elaborate on these cornerstones below.*
```

fingolfin replied:
There is still a link to https://github.com/flintlib/antic here, do you want that, @thofma ?

fingolfin committed:
Use callout syntax for caption

fingolfin commented on about.md:
```diff
@@ -1,91 +1,100 @@
+
+<img src="{{ site.baseurl }}/public/OSCAR-overview.png" alt="OSCAR Overview" width="70%" style="margin-left:13%;" align="center">
+
+{: .caption }
```
I've rewritten how this "caption" works, moving the CSS to its own file and using this "callout" syntax. The main advantage is that Jekyll applies its text filters to the text below, and e.g. turns `"OSCAR"` into `“OSCAR”` (with "smart quotes"). It also separates content from presentation.

fingolfin replied:
I've rewritten how this "caption" works, moving the CSS to its own file and using this "callout" syntax. The main advantage is that Jekyll applies its text filters to the text below, and e.g. turns `"OSCAR"` into `“OSCAR”` (with "smart quotes"). It also separates content from presentation.

fingolfin committed:
Apply suggestions from code review

fingolfin committed:
Rewrap content

fingolfin commented on about.md:
```diff
@@ -1,91 +1,162 @@
+  [GAP](https://www.gap-system.org),
+  [Singular](https://www.singular.uni-kl.de),
+  [Polymake](https://polymake.org), and the now-retired
+  [ANTIC](https://github.com/flintlib/antic),
```
so after my changes this now is the "antic" link which @thofma may or may not wish to replace

thofma commented on about.md:
```diff
@@ -1,91 +1,162 @@
+on the capabilities of four major systems:
+  [GAP](https://www.gap-system.org),
+  [Singular](https://www.singular.uni-kl.de),
+  [Polymake](https://polymake.org), and the now-retired
+  [ANTIC](https://github.com/flintlib/antic),
+whose functionality lives on through
+  [Hecke](https://github.com/thofma/Hecke.jl),
+  [Nemo](https://github.com/Nemocas/Nemo.jl), and
+  [AbstractAlgebra](https://github.com/Nemocas/AbstractAlgebra.jl).
```
```suggestion
  [Polymake](https://polymake.org), and
  ANTIC,
whose functionality is provided by
  [Hecke](https://github.com/thofma/Hecke.jl),
  [Nemo](https://github.com/Nemocas/Nemo.jl), and
  [AbstractAlgebra](https://github.com/Nemocas/AbstractAlgebra.jl).
```
Reformulations welcome

fingolfin replied:
Looks good to me

thofma commented:
I have to push the updated pdf and the png. How did you create the png?

fingolfin commented:
It would be good if the instructions for producing the PDF and PNG where in the repository; e.g. as a script `etc/regenerate-OSCAR-overview.sh`.

thofma committed:
Update about.md

HereAround committed:
Remove pdf of OSCAR overview picture, as not needed anymore

HereAround committed:
Add instructions and script to recreate the OSCAR overview picture, then update the overview picture

HereAround commented:
> It would be good if the instructions for producing the PDF and PNG where in the repository; e.g. as a script `etc/regenerate-OSCAR-overview.sh`.

@fingolfin @thofma I have just added a script (and corresponding instructions) which should make updates to the overview picture very easy in the future.

I have also updated the preview: https://herearound.github.io/oscar-website/about/

HereAround commented:
> I have to push the updated pdf and the png. How did you create the png?

I have just added a script to automatize these steps: `/etc/regenerate-OSCAR-overview.sh`

HereAround commented:
I believe we do not need the pdf version of OSCAR overview picture on the website. Please correct if you think differently. 

I have therefore removed the pdf from this PR. Also, my script will remove auxiliary files including the created pdf, so that execution of the script will only update the `public/OSCAR-overview.png` file.

thofma commented:
Thanks. This looks good from my side.
